src/database/queries/addUser.js

```diff
@@ -0,0 +1,3 @@
+const connction = require('../connection.js');
+const addUser = (user_name,password)=> connction.query(`INSERT INTO users(user_name,password)values($1,$2)`,[user_name,password]);
```

Comment:
@denaHS One more note that I forgot before - do you intend to hash the password in this function before storing it?  This might be the perfect place to do that password hashing.